xdsl/printer.py

```diff
@@ -226,12 +227,14 @@ def _print_results(self, op: Operation) -> None:
         self.print_list(op.results, self.print)
         self.print(" = ")
 
-    def print_ssa_value(self, value: SSAValue) -> None:
+    def print_ssa_value(self, value: SSAValue) -> str:
```

Comment:
can you please do this in a separate PR also, with associated tests? Probably there are existing tests to update

Reply:
#2377